cmd/adapter-rest/startcmd/start.go

```diff
@@ -635,6 +649,18 @@ func addIssuerHandlers(parameters *adapterRestParameters, ariesCtx ariespai.CtxP
 		return fmt.Errorf("failed to init storage provider : %w", err)
 	}
 
+	var governanceProvider *governance.Provider
+
+	if parameters.governanceVCSURL != "" {
+		var errNew error
+		// TODO add token parameter
+		governanceProvider, errNew = governance.New(parameters.governanceVCSURL, &tls.Config{RootCAs: rootCAs}, store,
+			map[string]string{})
+		if errNew != nil {
+			return errNew
+		}
+	}
+
```

Comment:
May be we can combine initialization of governanceProvider for issuer/rp in one place.

Reply:
done